PD - Adding intial pass of pages.

eciramella-r7 opened the pull request:
Trying to get some basic ideas and sections laid out for upcoming refresh of various tools documentation pages.

Commit:
PD - Adding intial pass of pages.

Please note, this is merely a concept.  Trying to get some basic ideas across.

fmitchell-r7 commented:
Yay! Thanks for kicking this off.

fmitchell-r7 commented on index.html:
```diff
@@ -1,2 +1,144 @@
+          <a class="navbar-brand" href="#">Rapid7 Open Source Toolkit - R.O.S.T</a>
+          <a class="navbar-brand" href="#security" style="color:#FF7F00"><i class="fa fa-shield"></i> Metasploit</a>
+          <a class="navbar-brand" href="#security" style="color:#FF7F00"><i class="fa fa-shield"></i> Security</a>
+          <a class="navbar-brand" href="#config" style="color:#497CE2"><i class="fa fa-cogs"></i> Property Management</a>
```
You might want to change "Property Management" to "Configuration Management" so it matches the heading below. Or visa versa.

Commit:
Cleaning up the index.html file

Making sure headders and related boxes have matching names.

fmitchell-r7 commented on index.html:
```diff
@@ -1,2 +1,144 @@
+        <div class="row">
+          <div class="col-md-8 col-md-offset-2 grow" style="border: 10px solid #497CE2;">
+            <h2><i class="fa fa-cogs"></i> Configuration Management</h2>
+            <h3 style="color:#497CE2">Something clever about config management.</h3>
```
Is this clever enough? "Leverage these tools to make live updates to running servers easier."

Commit:
Removing projects as requested

Rex, ruby_smb, and warvox were requested to be removed.

Commit:
Removing local copies of bootstrap

I don't necessarilly want to checkin anything that I can otherwise locate.

I mopped up the directories, but failed to preserve the files.

Fixed.

fmitchell-r7 commented:
👍 🍰 🎆 🚀 🚢 it.
